test/xbgpu/test_engine.py

```diff
@@ -177,6 +178,14 @@ def valid_end_to_end_combination(combo: dict) -> bool:
 class TestEngine:
     r"""Grouping of unit tests for :class:`.XBEngine`\'s various functionality."""
 
+    @pytest.fixture
+    def mock_send_stream_network(self) -> IPv4Network:
+        """Filter the output queues to just those corresponding to the --corrprods.
```

Comment:
@amishatishpatel, did you address this comment at all, perhaps during your real-time feedback session?

Reply:
I'd mentioned in my [overall comment](https://github.com/ska-sa/katgpucbf/pull/600#issuecomment-1624877017) before re-requesting a review that I haven't implemented changes/updates to `test_engine.py` just yet (it might take another chunk of time Re: Testing different `heap_accumulation_threshold`s).

The more [top-level aspect](https://github.com/ska-sa/katgpucbf/pull/600#discussion_r1246734554) of this has been addressed, abstracting the common (base?) fixture from fgpu and xbgpu into `test/conftest.py`.

Reply:
I'm not following why it's necessary to override the base implementation. Are there UDP send streams being created that you're trying to filter out? Or is it just future-proofing for when you add beamforming?

Reply:
Promise I haven't been ignoring this comment. Yes, largely for future-proofing reasons. Admittedly, I may have included this pre-emptively.